docs/design/architecture.md

```diff
@@ -218,8 +218,8 @@ the tunnel.
 
 <img src="../assets/service_walk.svg.png" width="600" alt="Antrea Service Traffic Walk">
 
-`kube-proxy` can be used in any supported mode: user-space iptables, or IPVS.
-See the [Kubernetes Service documentation](https://kubernetes.io/docs/concepts/services-networking/service)
+`kube-proxy` can be used in any supported mode: IPTables, IPVS or NFTables.
```

Comment:
nit: the K8s documentation seems to be using the following capitalization: iptables, IPVS, nftables
so maybe we should do the same?